dashboard/app.py

```diff
@@ -33,8 +33,6 @@
 opt_manager = None
 cal_manager = None
 
-# load database
-db = load_database()
```

Comment:
Removing this from the global module variables creates issues with a function below that relies on `db`.